recipe/meta.yaml

```diff
@@ -1,48 +1,44 @@
-{% set version = "4.4.1.1" %}
+{% set version = "4.5.0" %}
 
 package:
   name: libnetcdf
   version: {{ version }}
 
 source:
-  fn: v{{ version }}.tar.gz
-  url: https://github.com/Unidata/netcdf-c/archive/v{{ version }}.tar.gz
-  sha256: 7f040a0542ed3f6d27f3002b074e509614e18d6c515b2005d1537fec01b24909
+  url: https://github.com/Unidata/netcdf-c/archive/v{{ version }}-rc2.tar.gz
+  sha256: d5c858362730a849b704f187739ffff30f8cae44c59e4c509fa318e48e3229dc
   patches:
-    - dfile.c.patch  # [win]
-    - dim.c.patch  # [win]
-    - semantics.c.patch  # [win]
-    - dap.path  # [win]
-    - CMakeLists.patch  # [win]
+    - CMakeLists.patch
```

Comment:
Most of these patches are, in a way, upstream with the exception of `CMakeLists.patch` that add `find_package(Threads REQUIRED)` for `conda-forge`'s thread safe `HDF 5`.